addons/l10n_be_pos_sale/tests/test_l10n_be_pos_sale.py

```diff
@@ -13,6 +15,29 @@ def setUpClass(cls, chart_template_ref='l10n_be.l10nbe_chart_template'):
         super().setUpClass(chart_template_ref=chart_template_ref)
 
     def test_settle_order_is_invoice(self):
+
+        intracom_tax = self.env['account.tax'].create({
+            'name': 'test_intracom_taxes_computation_0_1',
+            'amount_type': 'percent',
+            'amount': 21,
+            'invoice_repartition_line_ids': [
+                (0, 0, {'repartition_type': 'base', 'factor_percent': 100.0}),
+                (0, 0, {'repartition_type': 'tax', 'factor_percent': 100.0}),
+                (0, 0, {'repartition_type': 'tax', 'factor_percent': -100.0}),
+            ],
+            'refund_repartition_line_ids': [
+                (0, 0, {'repartition_type': 'base', 'factor_percent': 100.0}),
+                (0, 0, {'repartition_type': 'tax', 'factor_percent': 100.0}),
+                (0, 0, {'repartition_type': 'tax', 'factor_percent': -100.0}),
+            ],
+        })
+
+        def _pos_data_process(self, loaded_data):
+
+            res = super(PosSession, self)._pos_data_process(loaded_data)
+            loaded_data['intracom_tax_ids'] = intracom_tax.ids
+            return res
```

Comment:
I think mocking it like this isn't great, because it means we're not testing the part of the code that we're supposed to test. Can't we just set the `intracom_tax` on the correct fiscal position?

Reply:
Well I spent so much time trying to make to follow the test from the other PR. I was having a hard time because intracom_fpos was a template and it was not possible to set an `account.tax` on it. i could set a template tax but then I wasn't able to add it on the sale order. 

Your above suggestion makes it work perfectly now thx